docs/members-example/reducers/MemberEditReducers.ts

```diff
@@ -0,0 +1,96 @@
+import * as _ from 'underscore';
+import { IReduxAction } from '../../../src/utils/ReduxUtils';
+import { defaultMemberAttributes } from '../models/Member';
+import { IMemberEditViewState } from '../views/MemberEditView';
+import {
+  MemberEditActionsType,
+  IMemberEditActionPayload,
+  IChangeMemberEmailPayload,
+  IChangeMemberSendEmailPayload,
+  IToggleMemberOpenPayload
+} from '../actions/MemberEditActions';
+
+export const defaultMemberEditViewState: IMemberEditViewState = {
+  appliedState: null,
+  editionState: defaultMemberAttributes,
+  id: null,
+  isOpen: false
+};
+
+const toggleMemberOpen = (state: IMemberEditViewState = defaultMemberEditViewState,
```

Comment:
instead of having `state: IMemberEditViewState = defaultMemberEditViewState` in every method, put it once in the `memberEditReducers` function